ccmlib/scylla_repository.py

```diff
@@ -567,3 +562,37 @@ def run_scylla_unified_install_script(install_dir, target_dir, package_version):
     run('''{0}/install.sh --prefix {1} --nonroot{2}'''.format(
         install_dir, target_dir, install_opt), cwd=install_dir)
     run(f'''ln -s {install_dir}/scylla/conf conf''', cwd=target_dir)
+
+
+Architecture = Literal['x86_64', 'aarch64']
+
+
+def get_manager_latest_reloc_url(branch: str = "master", architecture: Architecture = None) -> str:
```

Comment:
if we want to enforce type annotations, we might need to, well, enforce it. like

```py
def get_manager_latest_reloc_url(branch: str = "master", architecture: Optional[Architecture] = None) -> str:
```

Reply:
It's mostly for documentation now, not going to enforce it with tools.

Reply:
yes, if we want to use for document, i think we need to get it right from the very beginning.